books/free-programming-books-id.md

```diff
@@ -68,6 +69,7 @@
 
 ### HTML and CSS
 
+* [BELAJAR WEB PROGRAMMING: Referensi Pengenalan Dasar Tahapan Belajar Pemrograman Web Untuk Pemula](https://www.researchgate.net/publication/372959551_BELAJAR_WEB_PROGRAMMING_Referensi_Pengenalan_Dasar_Tahapan_Belajar_Pemrograman_Web_Untuk_Pemula) - Haris Kurniawan, S.Pd, M.Kom, dkk.
```

Comment:
This may not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.
Although the file is posted by one of the authors, it's not clear that this author has rights to the whole book.
we do not use honorifics such as "'S.Pd, M.Kom, dkk."

Reply:
got it will remove the mentioned book